FIAT/finite_element.py

```diff
@@ -137,10 +211,24 @@ def space_dimension(self):
         "Return the dimension of the finite element space."
         return self.poly_set.get_num_members()
 
-    def tabulate(self, order, points):
+    def tabulate(self, order, points, entity=None):
         """Return tabulated values of derivatives up to given order of
-        basis functions at given points."""
-        return self.poly_set.tabulate(points, order)
+        basis functions at given points.
+
+        Modified for trace element tabulation - passing information
```

Comment:
Previous version + changes => bad
Current version => good

If one wants to look at the history of a code base, there is the version control system. One of the signs of poorly maintained code base is when a smart programmer can reconstruct the story of the development of the software without looking at the version control, only by reading the current version. If a software is properly refactored as it grows, this will not be possible.

So, instead of appending the change, can you just update the documentation such that it accurately describes current behaviour?